### ISSUE TITLE
Atv, tvl, tv oost, rob-tv not working 

### ISSUE THREAD

**Issue by Dominiquedb** [add-ons/plugin.video.regiotv#44]
All the http://api.new.livestream.com/accounts/27755193/events/xxxxxxx/live.m3u8 streams are no longer working. 

Kodi is giving the message: afspelen mislukt, een of meer items kunnen niet worden afgespeeld. Controleer het log voor meer informatie over dit bericht.

When trying the link on a browser the message "{"name":"NotFoundError","message":"No live broadcast found."}" is given.

On the websites of Atv, tvl, tv oost, rob-tv livestreams are still available.

Atv does work on kanalenlijst hans.

**Comment by Bart80-1** [comment]
Just a question about this... am I right to assume that the live_stream URL's in /resources/lib need to be updated?
I think I was able to find out the correct API for one of the regional TV's (I only tried my local one) by taking the URL for the website (so no app involved). Not sure if this is the way to go...
I also only tried it on Kodi for Linux, not on a RPi.

**Comment by peno64** [comment]
Yes adapting the urls will probably do the job. I did it for atv and this one works. I created a pool request for atv but the developer needs to approve it...

**Comment by peno64** [comment]
@Bart80-1 If you provide me the urls I can try if they work and if so I will also create a pool request

**Comment by peno64** [comment]
I also fixed the urls for rob tv, tvl, tv oost
Hopefully the PR is accepted soon

**Comment by peno64** [comment]
The addon is updated for atv, rob tv, tvl, tv oost
New release is 0.1.5
